bin/control.bat

```diff
@@ -224,3 +183,20 @@ del %RESTART_SUCCESS_FILE%
 if not "%NO_PAUSE%" == "1" pause
 
 goto :eof
+
+:getJavaSpecificOpts
+if %1 == 8 (
+    set CONTROL_JVM_OPTS=-XX:+AggressiveOpts %2
+    exit /b 0
+)
+
+if %1 GEQ 9 if %1 LSS 11 (
+    set CONTROL_JVM_OPTS=-XX:+AggressiveOpts --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED --illegal-access=permit --add-modules=java.xml.bind %2
```

Comment:
Could you please list the options one per line? It is more readable and easier to review.